htlcswitch/link_test.go

```diff
@@ -5586,3 +5586,190 @@ func TestChannelLinkCanceledInvoice(t *testing.T) {
 		t.Fatalf("expected unknown payment hash, but got %v", err)
 	}
 }
+
+func TestChannelLinkHoldInvoiceSettle(t *testing.T) {
+	t.Parallel()
+
+	channels, cleanUp, _, err := createClusterChannels(
+		btcutil.SatoshiPerBitcoin*3,
+		btcutil.SatoshiPerBitcoin*5)
+	if err != nil {
+		t.Fatalf("unable to create channel: %v", err)
+	}
+	defer cleanUp()
+
+	n := newThreeHopNetwork(t, channels.aliceToBob, channels.bobToAlice,
```

Comment:
Folding style violation.

Reply:
Few others in this test as well, won't point out the rest to reduce spam.

Reply:
There's also a large portion of overlap between the tests that can likely be consolidated.

Reply:
Folding styles fixed.
Overlap consolidated.
Changed use of three hop network to two hop network, as the third hop wasn't used in the tests.